pkg/bbgo/order_executor_general.go

```diff
@@ -407,22 +407,32 @@ func (e *GeneralOrderExecutor) ClosePosition(ctx context.Context, percentage fix
 	atomic.AddInt64(&e.closing, 1)
 	defer atomic.StoreInt64(&e.closing, 0)
 
-	// check base balance and adjust the close position order
-	if e.position.IsLong() {
-		if baseBalance, ok := e.session.Account.Balance(e.position.Market.BaseCurrency); ok {
-			submitOrder.Quantity = fixedpoint.Min(submitOrder.Quantity, baseBalance.Available)
-		}
-		if submitOrder.Quantity.IsZero() {
-			return fmt.Errorf("insufficient base balance, can not sell: %+v", submitOrder)
+	if e.session.Futures { // Futures: Use base qty in e.position
+		submitOrder.Quantity = e.position.GetBase().Abs()
+		submitOrder.ReduceOnly = true
+		if e.position.IsLong() {
+			submitOrder.Side = types.SideTypeSell
+		} else {
```

Comment:
check e.position.IsShort() and avoid dust quantity?

Reply:
For futures, the dust position still can be closed if the ordeer is reduced only